rust/rope/src/compare.rs

```diff
@@ -13,9 +13,10 @@
 // limitations under the License.
 
 //! Fast comparison of rope regions, principally for diffing.
+#![allow(clippy::cast_ptr_alignment, clippy::unreadable_literal)]
```

Comment:
ditto keeping these attached to specific functions if possible; these lints are useful, and we only want to ignore them intentionally, case by case.